project/newEvaluation.html

```diff
@@ -0,0 +1,474 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>New Evaluations</title>
+
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+  <header>
+    <img src="assets/logo-v2.png" alt="Softvision Logo" class="sv-logo">
+    <nav style="margin-left: 20em; display:inline-block;">
+      <a style="font-size: x-large; color: #696969; text-decoration: none;" href="">Evaluations</a>
+      <a style="font-size: x-large; color:	#696969; text-decoration: none;" href="">New Evaluation</a>
+    </nav>
+    <a style="margin-left: 40%; display: inline-block; font-size: x-large; color: #696969; text-decoration: none;" href="">Log out</a>
+  </header>
+
+  <hr>
+
+  <form class="center">
+
+      <section>
+        <input type="text" placeholder="Candidate">
+        <input type="text" placeholder="Interviewer">
+        <input type="date" style="border: none">
+      </section>
+
+      <section>
+        <h2>Technical Level</h2>
+        <table style="margin-top: 0%">
+          <tr>
+            <th>Trainee</th>
+            <th>Junior</th>
+            <th>Middle</th>
+            <th>Senior</th>
+          </tr>
+          <tr class="border-lists">
+            <td>
+              <input type="radio" id="T"/>
+            </td>
+            <td>
+              <input type="radio" id="J1"/>
+              <input type="radio" id="J2"/>
+              <input type="radio" id="J3"/>
+            </td>
+            <td>
+              <input type="radio" id="M1"/>
+              <input type="radio" id="M2"/>
+              <input type="radio" id="M3"/>
+            </td>
+            <td>
+              <input type="radio" id="S1"/>
+              <input type="radio" id="S2"/>
+              <input type="radio" id="S3"/>
+            </td>
+          </tr>
+        </table>
+      </section>
+
+
+      <section>
+        <h2>Should the candidate be hired?</h2>
+        <textarea rows="6" placeholder="The type of project that is suitable for the candidate
+Is guidance required for the candidate" reqired></textarea>
+      </section>
+
+      <section>
+        <h2>General Impression</h2>
+        <textarea rows="6" required placeholder="*required" impressions"></textarea>
+      </section>
+
+      <section>
+        <h2>Workflow, Leadership & Soft Skills</h2>
+        <textarea rows="6" required placeholder="Describe the environment in which the candidate works.
+    Describe any guidance or management experience" workflow"></textarea>
+      </section>
+
+      <fieldset>
+        <legend>OOP, Design Patterns</legend>
+        <ul>
+          <li>
+            <label for="classes">Classes</label>
+            <select name="classes" id="classes">
+              <option value="0">0</option>
```

Comment:
I'd normally say something purposely misleading along the lines of "Ah, if only there was a way of doing this such that we didn't have to repeat all of this markup...", but @zuldy would probably slap me silly if I pulled that sort of stunt with you guys again... :P We're gonna get to JavaScript soon, and we'll see nicer ways of generating repeatable pieces of code.